### ISSUE TITLE
Helios link in README is 404

### ISSUE THREAD

**Issue by DemiMarie** [oxidecomputer/helios-omicron-brand#7]
The readme links to <https://github.com/oxidecomputer/helios>, but that is 404 for me.

**Comment by jclulow** [comment]
Yes!  Sorry about that.  I have a little clean-up to do there and I'll get it opened soon.

**Comment by DemiMarie** [comment]
> Yes! Sorry about that. I have a little clean-up to do there and I'll get it opened soon.

Thank you!
